lib/Doctrine/DBAL/Platforms/SQLServerPlatform.php

```diff
@@ -169,6 +169,20 @@ public function getDropDatabaseSQL($name)
     }
 
     /**
+     * Returns the SQL statement for closing currently active connections on the given database.
+     *
+     * This is useful to force DROP DATABASE operations which could fail because of active connections.
+     *
+     * @param string $database The name of the database to close currently active connections for.
+     *
+     * @return string
+     */
+    public function getCloseActiveDatabaseConnectionsSQL($database)
```

Comment:
I get your point but I do not want to enforce an abstraction here because it probably will be too vendor specific. Another option would be to put the SQL directly into the `dropDatabase()` method in the schema manager.